### PR TITLE
Add projects to user profile

### LINKED ISSUE #88: Use Jinja comments for copyright header in templates
Currently the generated HTMLs are polluted with inaccurate copyright headers (in the sense that generated data are not copyrighted).  Instead, we should use [Jinja comments](https://jinja.palletsprojects.com/en/2.11.x/templates/#comments) to specify the copyright only in the templates, e.g.

```jinja
{#
 # Base HTML template
 # Copyright (C) ...
 #
 # This file is part of Acanban.
 #
 # Acanban is free software: you can redistribute it and/or modify
 # it under the terms of the GNU Affero General Public License as published
 # by the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
 #
 # Acanban is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Affero General Public License for more details.

 # You should have received a copy of the GNU Affero General Public License
 # along with Acanban.  If not, see <https://www.gnu.org/licenses/>.
 #}
```

### LINKED ISSUE #107: Add projects into user's profile

### PR DESCRIPTION
**Pull request by Huy-Ngo**
This closes #107 

I also resolve #88 here.

### CODE REVIEW

File: src/acanban/user.py [hunk 1]
```diff
@@ -40,8 +40,11 @@ async def view_user_profile(username: str) -> ResponseReturnValue:
     """Handle the user profile page."""
     async with current_app.db_pool.connection() as conn:
         user = await users.get(username).run(conn)
-    if user is None: raise NotFound
-    return await render_template('user.html', user=user)
+        if user is None: raise NotFound
+        project_uuids = user['projects']
```
Comment (McSinyx) [inline src/acanban/user.py:44]: Currently not all users may have the field `projects`, so `get('projects', ...)` should be used here instead, where … could either be None or [] depending on your preference.

Comment (Huy-Ngo) [reply src/acanban/user.py:44]: like `user.get('projects', None)`?

File: src/acanban/templates/user.html [hunk 2]
```diff
@@ -37,5 +38,18 @@
     <button>Edit profile</button>
   </a>
   {% endif %}
+  {% if projects %}
+  <h1>{{ user.name }}'s project(s)</h1>
```
Comment (McSinyx) [inline src/acanban/templates/user.html:42]: This should be indented.

Comment (Huy-Ngo) [reply src/acanban/templates/user.html:42]: Vim unindented it so I thought it's supposed not to be :<

Comment (Huy-Ngo) [reply src/acanban/templates/user.html:42]: Also now that I understand why some devs are too lazy to pluralize properly :v.

Comment (McSinyx) [reply src/acanban/templates/user.html:42]: My bad, Vim is God :smile:  Jokes aside you're right, the part below has to be dedented (h1 and the listing are fof the same level, my eyes got tricked).

**Comment by Huy-Ngo** [comment]
To pluralize properly, I can either enable  [Jinja i18n extension](https://jinja.palletsprojects.com/en/2.11.x/templates/) or install [another package](https://pypi.org/project/jinja2_pluralize/) and I'm not sure if it's worth it.

File: src/acanban/templates/user.html (hunk 2 above)
Comment (Huy-Ngo) [reply src/acanban/templates/user.html]: Oh so I should dedent all of them now.

Comment (McSinyx) [reply src/acanban/templates/user.html:42]: Yep, sorry.

**Comment by McSinyx** [comment]
To be *that guy*, it's an iterable, not a sequence (which needs `__len__`).  Personally I'd prefer keeping it as always plural over the (s) if it's not trivial to be done.

**Review by McSinyx — APPROVED** [review]
We are [on the `net/`](https://www.kernel.org/doc/html/latest/process/coding-style.html#commenting) now!

**Comment by Huy-Ngo** [comment]
I receive these messages errors when I try to treat it as such:

```
  File "/home/xarvos/Quell/acanban/src/acanban/templates/user.html", line 41, in block "content"
    {% if projects|length > 0 %}
TypeError: object of type 'TrioCursor' has no len()
```

```
  File "/home/xarvos/venvs/acanban/lib/python3.8/site-packages/acanban/user.py", line 47, in view_user_profile
    return await render_template('user.html', user=user, projects=list(projects))
TypeError: 'TrioCursor' object is not iterable
```

**Comment by McSinyx** [comment]
Ah it's an AsyncIterable, so `__iter__` is not implemented by `__aiter__` (and thus the simulated `for` loop in `list` wouldn't work, you'd need `async for`).

**Review by DDHLong — APPROVED** [review]

**Pull request merged**